tests/integration/iceberg/test_table_load.py

```diff
@@ -24,7 +24,7 @@ def test_daft_iceberg_table_open(local_iceberg_tables):
 
 
 WORKING_SHOW_COLLECT = [
-    "test_all_types",
+    # "test_all_types",
```

Comment:
We should leave a comment on why we're skipping this test, and create a corresponding github issue for the broken test

Reply:
3b5b3972a553642b68a5206eb02f42badbc4521d